src/Slider/Slider.js

```diff
@@ -581,13 +598,20 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	updatePopupPosition: function (percent) {
+	updateKnobPosition: function (percent) {
 		this.$.knob.applyStyle(this.get('orientation') == 'vertical' ? 'bottom' : 'left', percent + '%');
 	},
 
 	/**
 	* @private
 	*/
+	updatePopupPosition: function () {
+		;	// Do not allow ProgressBar to update popup position
```

Comment:
I don't think empty methods is a practice that we endorse. Can we find another way to do this?